src/sql/workbench/parts/notebook/models/cell.ts

```diff
@@ -393,9 +396,10 @@ export class CellModel implements ICellModel {
 					if (model.activeConnection) {
 						let endpoint = this.getKnoxEndpoint(model.activeConnection);
 						let host = endpoint && endpoint.ipAddress ? endpoint.ipAddress : model.activeConnection.serverName;
+						let port = endpoint && endpoint.port ? ':' + endpoint.port.toString() : defaultPort;
 						let html = result.data['text/html'];
-						html = this.rewriteUrlUsingRegex(/(https?:\/\/mssql-master.*\/proxy)(.*)/g, html, host);
-						html = this.rewriteUrlUsingRegex(/(https?:\/\/master.*master-svc.*\/proxy)(.*)/g, html, host);
```

Comment:
please be careful here. we had 2 for backwards compatibility (CTP 2.4 and older). Would rather add to this list instead of replace here.